app/__init__.py

```diff
@@ -30,5 +30,10 @@ def create_app(test_config=None):
     migrate.init_app(app, db)
 
     # Register Blueprints here
+    from .routes import task_bp
+    app.register_blueprint(task_bp)
+
+    from .routes import goal_bp
+    app.register_blueprint(goal_bp)
```

Comment:
you can save a few lines of code by combining the imports 
```suggestion
    from .routes import task_bp, goal_bp
    app.register_blueprint(task_bp)
    app.register_blueprint(goal_bp)
```